.github/workflows/full-android.yaml

```diff
@@ -73,30 +73,6 @@ jobs:
                     "./scripts/build/build_examples.py --target android-arm-chip-tool build"
             - name: Clean out build output
               run: rm -rf ./out examples/android/CHIPTool/app/libs/jniLibs/* examples/android/CHIPTool/app/libs/*.jar
-            - name: Build Android arm-tv-casting-app
-              run: |
-                  ./scripts/run_in_build_env.sh \
-                    "./scripts/build/build_examples.py --target android-arm-tv-casting-app build"
```

Comment:
Why is this being removed?  Nothing else builds android-arm-tv-casting-app or android-arm-tv-server....  The "examples" thing mentioned only builds the arm64 versions.

Reply:
This is true ... it is likely worth it to build arm as it is 32-bit so int/long conversions will be different than the 64-bit builds.